scripts/metadata-get.mjs

```diff
@@ -16,8 +16,8 @@ const CMD = {
 };
 
 // V15
-const META_VERSION_HEX = '0x0f000000';
-const META_VERSION = 15;
+const META_VERSION_HEX = '0x10000000';
```

Comment:
I would suggest we only change this when its time to get the new metadata in a stable sdk release. That way for every release before that happens we can still get metadata v15 and update it. Thoughts?